### PR TITLE
ops: flattener command

### PR DESCRIPTION
**Pull request by ivaylonikolov7**
This is the flattener command that removes the unnecessary duplicated headings. Also adds image button to the readme. We can change the layout of the README.md. The place I put that button is just for example. I wasnt sure where's the correct place.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+  flatten:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Install Hardhat
+        run: npm install hardhat
+      - name: Run flattener
+        run: bash bin/flattener.sh
```
Comment (imollov) [inline .github/workflows/main.yml:18]: Is it going to commit the generated file in the repo?

File: README.md [hunk 2]
```diff
@@ -44,3 +44,9 @@ npx hardhat verify --network ropsten DEPLOYED_CONTRACT_ADDRESS "Hello, Hardhat!"
 # Performance optimizations
 
 For faster runs of your tests and scripts, consider skipping ts-node's type checking by setting the environment variable `TS_NODE_TRANSPILE_ONLY` to `1` in hardhat's environment. For more details see [the documentation](https://hardhat.org/guides/typescript.html#performance-optimizations).
+
+[![Remix image][image]][hyperlink]
```
Comment (imollov) [inline README.md:48]: Let's leave only the link for now.

File: bin/flattener.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```
Comment (imollov) [inline bin/flattener.sh:2]:
```suggestion
```

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```
Comment (imollov) [inline .github/workflows/main.yml:7]: Why on pull request?

File: .github/workflows/main.yml (hunk 1 above)
Comment (imollov) [reply .github/workflows/main.yml:18]: Also, please add it as an npm command.

**Comment by imollov** [comment]
@ivaylonikolov7 as discussed, please rename the flattened file and path in a more descriptive and conventional way.

**Comment by ivaylonikolov7** [comment]
> @ivaylonikolov7 as discussed, please rename the flattened file and path in a more descriptive and conventional way.

Sorry, I forgot what would be a better descriptive name. I checked beefyfinance and they have tmp too.

**Pull request merged**